### PR TITLE
add new program

### PR DESCRIPTION
**Pull request by tati**

### CODE REVIEW

File: bootcamps/thinkful/data.yml [hunk 1]
```diff
@@ -354,6 +354,63 @@ programs:
...
+    - seattle
+    - tampa
+    - washington-dc
+    commitment: part-time
+    cost_description: $1,495 per month
+    description: !!python/unicode 'The Data Analytics program combines an in-depth, 
+      project-based approach with two 1-on-1 sessions every week with a professional 
+      Data Analyst. This program is built to help those with no prior experience in 
+      data or technology land their first full time job as a data analyst - at a 
+      full-time immersive pace! You''ll work side-by-side
```
Comment (tati) [inline bootcamps/thinkful/data.yml]: this '' is not a typo, it is escaping

File: bootcamps/thinkful/data.yml [hunk 2]
```diff
@@ -354,6 +354,63 @@ programs:
...
+    - phoenix
+    - raleigh
+    - salt-lake-city
+    - san-diego
+    - san-francisco
+    - seattle
+    - tampa
+    - washington-dc
+    commitment: part-time
+    cost_description: $1,495 per month
```
Comment (mariahhoward) [inline bootcamps/thinkful/data.yml]: Monthly for DA-I is $3,375 per month

File: bootcamps/thinkful/data.yml [hunk 3]
```diff
@@ -354,6 +354,63 @@ programs:
...
+    description: !!python/unicode 'The Data Analytics program combines an in-depth, 
+      project-based approach with two 1-on-1 sessions every week with a professional 
+      Data Analyst. This program is built to help those with no prior experience in 
+      data or technology land their first full time job as a data analyst - at a 
+      full-time immersive pace! You''ll work side-by-side 
+      with a professional data analyst to learn statistics, analysis and experiment design, 
+      how to work with visualizations in Tableau, a thorough understanding of Excel, SQL and 
+      Python. You''ll also have the option to meet in person with fellow students and mentors 
+      to share ideas and work through problems.
```
Comment (mariahhoward) [inline bootcamps/thinkful/data.yml]: Recommended copy changes: 
Thinkful’s full-time Data Analytics Immersion shares the same in-depth, project-based approach of the Flexible pace. The main difference is that the Full Time pace pushes students to become data analysts in 4 months and requires over 40 hours of effort per week. The fully online program features constant support, as students interact with their classmates, instructors, and mentors. Each student has a dedicated mentor, who they meet with daily 1-on-1 for half an hour.

In the course, you will learn statistics, analysis and experiment design, how to work with visualizations in Tableau, a thorough understanding of Excel, SQL and Python.

File: bootcamps/thinkful/data.yml [hunk 4]
```diff
@@ -354,6 +354,63 @@ programs:
...
+      
+      '
+    display_name: Data Analytics Immersion
+    duration: 6
```
Comment (mariahhoward) [inline bootcamps/thinkful/data.yml]: duration should be 4 months

File: bootcamps/thinkful/data.yml [hunk 5]
```diff
@@ -354,6 +354,63 @@ programs:
...
+      within 6 months of graduating.
+    reports_outcomes: true
+    scholarships: For women and veterans
+    topics:
+    - mysql
+    - python
+    - sql
+    tuition: 
+    tuition_units: USD
   flexible-product-design-bootcamp:
```
Comment (mariahhoward) [inline bootcamps/thinkful/data.yml]: I'm not sure what line 414 is referring to but it looks like it should immersion-data-analytics-bootcamp

**Pull request merged**